src/main/scala/machines/Internal.scala

```diff
@@ -3,4 +3,31 @@ package machines
 import regex._
 import dfa._
 
-// TODO: Add your code below
+given Conversion[Char, RegularLanguage] =
+    c => Character(c)
+
+given Conversion[String, RegularLanguage] = 
+    s => s.map(c => Character(c)).reduce(Concat(_,_))
```

Comment:
Great use of higher order functions, it makes the code very minimal compared to the pattern matching approach I took!